slack/web/base_client.py

```diff
@@ -82,8 +84,9 @@ def _get_headers(self, has_json, has_files, request_specific_headers):
         # Merge headers specified at client initialization.
         final_headers.update(self.headers)
 
-        # Merge headers specified for a specific request. i.e. oauth.access
-        final_headers.update(request_specific_headers)
+        # Merge headers specified for a specific request. e.g. oauth.access
+        if request_specific_headers:
```

Comment:
Would you mind adding at least one test that hit this line? It would be really helpful to safeguard against future changes.

Reply:
@RodneyU215 the best way to test for this is via lint, not via unit tests. You should never set function arguments (like here: https://github.com/slackapi/python-slackclient/pull/555/files#diff-385cd56c2ef54a16a374209282ee8d3dR109) to mutable structures like {}, [], etc.